src/effects/effectchainslot.cpp

```diff
@@ -447,6 +449,11 @@ void EffectChainSlot::replaceEffect(unsigned int effectSlotNumber,
     if (!pEffect.isNull()) {
         if (m_bAddedToEngine) {
             qDebug() << "Adding " << pEffect->getManifest()->id() << " at " << effectSlotNumber;
+            qDebug() << "INPUT CHANNELS: ";
+            for (auto channel : m_enabledInputChannels) {
```

Comment:
m_enabledInputChannels should be removed. It is redundant with the ChannelInfos' ControlObjects.